packages/socket-stream-client/browser.js

```diff
@@ -114,8 +113,8 @@ export default class ClientStream extends StreamClientCommon {
   }
 
   _heartbeat_timeout() {
-    Meteor._debug('Connection timeout. No sockjs heartbeat received.');
-    this._lostConnection(new DDP.ConnectionError('Heartbeat timed out'));
```

Comment:
It's a shame this custom error type used a special `errorType` property.

It seems like we mainly check for this type of error internally, notably [`tools/meteor-services/auth-client.js#L33`](https://github.com/meteor/meteor/blob/devel/tools/meteor-services/auth-client.js#L33) and a couple places in [`tools/packaging/catalog/catalog.js`](https://github.com/meteor/meteor/blob/devel/tools/packaging/catalog/catalog.js#L126), but this presumably bubbles into those under some circumstances? Perhaps those should be updated?

Other than that, I did some GitHub [code searching](https://github.com/search?p=1&q=%22DDP.ConnectionError%22+extension%3Ajs+-filename%3Astream_client%2A+-filename%3Aservice-connection+-path%3Apackages%2Fddp-client+-path%3Aprograms%2F+-path%3Atests%2F+-filename%3Addp-client.js+-filename%3Aspec+-filename%3Astubs+-filename%3Aauth-client.js+-filename%3Acatalog.js&type=Code&utf8=%E2%9C%93) to try to hone in on any potential problems, but didn't come across anything concerning or that might be a breaking change, though I suppose developers could be looking for this type of error in their own private code?